### ISSUE TITLE
InlineFormAdmin ignores form_extra_fields

### ISSUE THREAD

**Issue by polarisary** [flask-admin/flask-admin#1802]
i write form as follows, "form_label" and "form_columns" works regular， but "form_extra_fields" does not work. Waiting, Thanks!
========================
class FollowupForm(InlineFormAdmin):
    form_label='Follow Up'
    form_columns=[
        'id', 'subject', 'template', 'body_html', 
        'body_text', 'level', 'reply'
     ]
    form_extra_fields={
        'subject': StringField('Subject'),
        'template': StringField('Template'),
        'body_html': TextAreaField('Content（HTML）',),
        
    }

**Comment by maksimartemevalar** [comment]
Bump

**Comment by jbbqqf** [comment]
Hi! Triaging older issues — the underlying limitation here is real but `form_extra_fields` was never the supported way to add fields to a Peewee inline form. The supported path goes through `InlineFormAdmin.postprocess_form`, which until very recently was not even invoked on the Peewee backend.

Evidence:
- Peewee's inline `InlineModelConverter.contribute` builds child forms by calling wtfpeewee's [`model_form(...)`](https://github.com/pallets-eco/flask-admin/blob/master/flask_admin/contrib/peewee/form.py#L320-L329), passing through only `form_columns`/`form_args` and ignoring `form_extra_fields`. So the original symptom isn't a bug in `form_extra_fields` handling for inlines -- it just isn't wired in that path on the Peewee backend.
- The supported hook for adding extra fields to an inline form is `InlineFormAdmin.postprocess_form`. On the Peewee backend that hook was *also* never called (a pre-existing gap relative to SQLAlchemy), so the docstring-recommended workflow was effectively no-op. I've sent a fix in **#2901** that (a) corrects the misleading `inline_model_form_converter` docstring and (b) wires up `info.postprocess_form(child_form)` inside Peewee's `contribute` so it matches SQLA's long-standing behavior.

Once #2901 lands, the supported way to get the original ticket's outcome is:

```python
from flask_admin.model.form import InlineFormAdmin
from wtforms import StringField

class FollowupInlineForm(InlineFormAdmin):
    form_label = "Follow Up"
    form_columns = ("id", "subject", "template", "body_html", "body_text", "level", "reply")

    def postprocess_form(self, form_class):
        form_class.my_extra = StringField("Extra")
        return form_class
```

If you agree, would you mind closing this in favour of tracking the Peewee inline fix in #2901? If you'd prefer to keep this open as a separate "support `form_extra_fields` on Peewee inline forms" request, that's also fine -- happy to be wrong if I missed an existing code path.

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the references above were verified manually.*
